platform/viewer/public/config/default.js

```diff
@@ -13,6 +13,7 @@ window.config = {
         qidoSupportsIncludeField: true,
         imageRendering: 'wadors',
         thumbnailRendering: 'wadors',
+        enableStudyLazyLoad: true,
```

Comment:
This new config allows to enable/disable the QIDO+WADO study metadata load. Setting this to false will make the viewer behave exactly as before (a single WADO call for the entire study metadata);

Reply:
I'm struggling to think of a reason we would need to preserve the old behavior.

Reply:
Maybe if QIDO-RS for study isn't supported, or if it is worse for performance on some servers. I would keep the flag but enable it by default.

Reply:
Yes. I think the performance gain can only be noticed if the user has predominantly very large studies with many series. If the user only has studies with one or two series I'd advise to disable it as the performance will probably be worse since we'll be essentially replacing a single API call by two... :-/